.github/workflows/ci.yml

```diff
@@ -72,6 +72,20 @@ jobs:
 
     runs-on: ubuntu-latest
 
+    services:
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_PASSWORD: password 
+          POSTGRES_DB: test
+        ports:
+          - 5432:5432
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
```

Comment:
Would it be possible/easier to boot up postgres from the `ironfish-http-api-v2/docker-compose.yml` file, to avoid any issues from configuration differences there?